cmd/cli/uninstall_mesh_test.go

```diff
@@ -268,6 +286,16 @@ func TestUninstallCmd(t *testing.T) {
 
 				if test.userPromptsYes {
 					if test.meshExists {
+						if len(test.nsInMesh) != 0 {
```

Comment:
Optional:

Using too many if...else is not preferred. Since removing these labels and annotation applies to the whole uninstall command, we can make the test case more general.

i.e.

```
tests := []struct {
	...
	workloadNamespaces        []string
	...
}

// when setting up test env
for workloadNs := range test.workloadNamespaces {
	// create workload namespace
	nsSpec := createNamespaceSpec(workloadNs, test.meshName, true, false, true)
	_, err = fakeClientSet.CoreV1().Namespaces().Create(context.TODO(), nsSpec, metav1.CreateOptions{})
	assert.Nil(err)
}

// verifying in test case (line 289)
for workloadNs := range test.workloadNamespaces {
	// check if labels and annotations are removed for workloadNs, same as your implementation from L290-L298
}
```

Reply:
@allenlsy I have implemented the requested change, PTAL